tests/python/contrib/test_hexagon/test_benchmark_elemwise_add.py

```diff
@@ -124,6 +119,7 @@ def _get_irmod_elemwise_add(
         # Also: The VTCM budget is a very rough estimate, based only on experience.
         # Assuming that it's even reasonable to use a hard-coded estimate AT ALL, this number
         # may need tweaking.
+        # pylint: disable=unreachable
```

Comment:
Can we not just remove this and add it when it's functional?

Reply:
Yeah I wasn't sure if this needs to be available as the author seems to have left it intentionally unreachable, but I've commented it out for now and mentioned that it can be enabled when needed.